frontend/src/APIClients/AuthAPIClient.ts

```diff
@@ -126,12 +111,49 @@ const refresh = async (): Promise<boolean> => {
   }
 };
 
+/**
+ * Retrieve the firebase user given a user id
+ * @param uid user's id
+ * @returns User: a firebase user instance
+ * @throws error if could not retrieve user by uid
+ */
+const getFirebaseUserByUid = async (uid: string): Promise<User> => {
+  const { data } = await baseAPIClient.get(`/auth/${uid}`, {
+    withCredentials: true,
+  });
+  return data;
+};
+
+/**
+ * Sets the verify status of a user to true on firebase
+ * @param uid user's id to identify record in firebase
+ * @returns true if able to verify the user by uid
+ * @throws error if could not verify user by uid
+ */
+const verifyEmail = async (uid: string): Promise<boolean> => {
+  const bearerToken = `Bearer ${getLocalStorageObjProperty(
+    AUTHENTICATED_USER_KEY,
+    "accessToken",
+  )}`;
+  try {
+    await baseAPIClient.post(
+      `/auth/verifyEmail/${uid}`,
+      {},
+      { headers: { Authorization: bearerToken } },
```

Comment:
Hmm do you still need to pass bearerToken since you don't have any middleware for authorization?